pagopa/io-app
# feat: [IOBP-70] Addition of idpay payment auth page

## Short description
added IDPay's payment auth page, along with its loading skeleton
<img width="200" alt="Screenshot 2023-06-09 at 15 00 26" src="https://github.com/pagopa/io-app/assets/60693085/2651218c-0f2a-4030-9251-5886532fd754">

## List of changes proposed in this pull request
- added page
- added skeleton

## How to test
with `Io-dev-server`, navigate to a discount initiative, click the "make a payment" CTA, use the code `00000001` and make sure everything renders correctly.
